vercel-request-handler/src/index.ts

```diff
@@ -21,10 +21,45 @@ app.get("/*", async (req, res) => {
         Key: `dist/${id}${filePath}`
     }).promise();
     
-    const type = filePath.endsWith("html") ? "text/html" : filePath.endsWith("css") ? "text/css" : "application/javascript"
+    const ext = filePath.split(".").pop();
+    let type: string;
+    switch (ext) {
+      case "html":
```

Comment:
Instead of doing this i'd recommend using a package like [mime-types](https://www.npmjs.com/package/mime-types) As it would support a lot of different files